src/results/electric_storage.jl

```diff
@@ -32,7 +32,8 @@ function add_electric_storage_results(m::JuMP.AbstractModel, p::REoptInputs, d::
         r["storage_to_load_series_kw"] = round.(value.(discharge), digits=3)
 
         r["initial_capital_cost"] = r["size_kwh"] * p.s.storage.attr[b].installed_cost_per_kwh +
-            r["size_kw"] * p.s.storage.attr[b].installed_cost_per_kw
+            r["size_kw"] * p.s.storage.attr[b].installed_cost_per_kw +
+            p.s.storage.attr[b].installed_cost_constant
```

Comment:
will this report a capital cost if the recommended storage size is zero?

Reply:
I could be misunderstanding the code, but I believe that the "if r["size_kwh"] != 0" expression on line 27 ensures that the r[“initial_capital_cost”] output only gets generated if the battery size is > 0 kWh. So if the battery kWh = 0, then r[“initial_capital_cost”] would not be created, so the installed cost constant would be not included.